### PR TITLE
Fix typo in help string for --fields option

### PR DESCRIPTION
**Pull request by balta2ar**
@masatake As discussed here: https://github.com/universal-ctags/ctags/commit/9135e21d4833d104ddf50b7fbc0a77ab9d21e766#commitcomment-18866858

### CODE REVIEW

File: main/options.c [hunk 1]
```diff
@@ -247,7 +247,7 @@ static optionDescription LongOptionDescription [] = {
  {1,"      Include selected extension fields (flags: \"afmikKlnsStzZ\") [fks]."},
  {1,"  --fields-<LANG|*>=[+|-]flags"},
  {1,"      Include selected <LANG> own extension fields"},
- {1,"      (flags: --list-flags=<LANG>)."},
+ {1,"      (fields: --list-fields=<LANG>)."},
```
Comment (masatake) [inline main/options.c]: Thank you.

I think `flags: --list-fields=<LANG>` is better. Here `flags` means the name of parameter of `--fields-` option.

Comment (balta2ar) [inline main/options.c]: @masatake Done!

**Comment by masatake** [comment]
Can I ask you to squash the two commits into one?

**Comment by balta2ar** [comment]
@masatake Sure!

**Comment by masatake** [comment]
Thank you.

**Pull request merged**